### PR TITLE
Add last_sth_timestamp to log signer

### PR DESCRIPTION
**Pull request by gdbelvin**
### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: log/sequencer.go [hunk 1]
```diff
@@ -80,6 +81,7 @@ func createMetrics(mf monitoring.MetricFactory) {
 	quota.InitMetrics(mf)
 	seqBatches = mf.NewCounter("sequencer_batches", "Number of sequencer batch operations", logIDLabel)
 	seqTreeSize = mf.NewGauge("sequencer_tree_size", "Size of Merkle tree", logIDLabel)
+	seqTimestamp = mf.NewGauge("sequencer_timestamp", "Time of last STH in ms since epoch", logIDLabel)
```
Comment (Martin2112) [inline log/sequencer.go]: Naming the new one `sequencer_tree_timestamp` would indicate that they're related.

Doc comment should probably say "last STH produced" or "signed" for clarity. Might also want to document both of those as only valid if master for that tree.

**Comment by pavelkalinnikov** [comment]
Is "STH" purely a CT term? Rename it to Trillian terms if so.

File: log/sequencer.go (hunk 1 above)
Comment (gdbelvin) [reply log/sequencer.go]: Added documentation suffix: "last SLR signed". 

All the metrics in this file are only valid for when master so I've omitted that clarification for consistency.

File: log/sequencer.go [hunk 2]
```diff
@@ -417,13 +419,15 @@ func (s Sequencer) IntegrateBatch(ctx context.Context, tree *trillian.Tree, limi
...
 		// Create the log root ready for signing
-		seqTreeSize.Set(float64(merkleTree.Size()), label)
 		newLogRoot = &types.LogRootV1{
 			RootHash:       merkleTree.CurrentRoot(),
 			TimestampNanos: uint64(s.timeSource.Now().UnixNano()),
 			TreeSize:       uint64(merkleTree.Size()),
 			Revision:       uint64(newVersion),
 		}
+		seqTreeSize.Set(float64(newLogRoot.TreeSize), label)
+		seqTimestamp.Set(float64(time.Duration(newLogRoot.TimestampNanos)*time.Nanosecond/
```
Comment (Martin2112) [inline log/sequencer.go]: `TimestampNanos` is already in nanos and `Duration` is documented as a nanosecond count. So you can remove the multiplier if you like.

**Pull request merged**